polygeist/lib/Dialect/Polygeist/Analysis/MemoryAccessAnalysis.cpp

```diff
@@ -535,8 +535,11 @@ getOffset(const Value expr, const SmallVectorImpl<Value> &loopAndThreadVars,
 // MemoryAccessMatrix
 //===----------------------------------------------------------------------===//
 
-[[maybe_unused]] raw_ostream &
-mlir::polygeist::operator<<(raw_ostream &os, const MemoryAccessMatrix &matrix) {
+namespace mlir {
+namespace polygeist {
```

Comment:
Using namespace qualifiers is what the [LLVM coding standards](https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions) say, so I think we should stick to that

Reply:
Yes I know but that doesn't work and g++ reports a warning. I changed it to get rid of the warning.